File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/InlineResponse2006.scala

package org.openapitools.client.model


case class InlineResponse2006 (
    _dataInputs: Option[List[DataInputHistoryResponse]]
)
object InlineResponse2006 {
    def toStringBody(var_dataInputs: Object) =
        s"""
        | {
        | "dataInputs":$var_dataInputs
        | }
        """.stripMargin
}

File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/InlineResponse2006.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.DataInputHistoryResponse
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param dataInputs 
 */
data class InlineResponse2006 (

        @JsonProperty("dataInputs") val dataInputs: List<DataInputHistoryResponse>? = null
) {

}

